public/static/styles/app.css

```diff
@@ -1,3 +1,5 @@
+@import 'add-to-calendar-button/assets/css/atcb.min';
```

Comment:
We tend to import libraries CSS from https://github.com/opencollective/opencollective-frontend/blob/main/pages/_app.js#L17